### PR TITLE
Updated translation

### PR DESCRIPTION
**Pull request by ognjenm**
Added missing validation strings from laravel 5

### CODE REVIEW

File: me/validation.php [hunk 1]
```diff
@@ -73,10 +75,11 @@
...
     ],
-    'string'               => 'The :attribute must be a string.',
-    'timezone'             => 'The :attribute must be a valid zone.',
-    'unique'               => 'Vrijednost za polje :attribute već postoji.',
-    'url'                  => 'Format polja :attribute ne važi.',
+    'string'           => 'Polje :attribute mora biti string.',
+    'timezone'         => 'The :attribute must be a valid zone.',
+    'unique'           => 'Vrijednost za polje :attribute već postoji.',
+    'url'              => 'Format polja :attribute ne važi.',
+    'pdv'              => 'Format polja PDV nije ispravan. Format je npr: 20/31-00112-1',
```
Comment (caouecs) [inline me/validation.php]: Hello,
*pdv* is not in official file

**Comment by caouecs** [comment]
Hello,
I updated files with a new keyword in validation ( *in_array* ).

Can you resolve conflicts and translate *in_array* ?

**Comment by ognjenm** [comment]
Hi 'pdv' string removed, in_array added

**Comment by caouecs** [comment]
This branch has conflicts

**Comment by thandonguocmo2020** [comment]
I want to contribute but do not know what to do. i is language vi. We wish to contribute.

**Comment by thandonguocmo2020** [comment]
`

`

**Comment by caouecs** [comment]
Hello @thandonguocmo2020,
you want to help @ognjenm with Ukrainian translation or on Vietnamese ( vi ) translation ?

**Comment by caouecs** [comment]
Thanks

**Comment by thandonguocmo2020** [comment]
I want to contribute to this language vi. guide me !

2016-03-11 14:49 GMT+07:00 Fred Delrieu <notifications@github.com>:

> Thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caouecs/Laravel-lang/pull/512#issuecomment-195238899>.
>



-- 
Thandonguocmo2020@gmail.com

**Comment by thandonguocmo2020** [comment]
thank you ! :)

2016-04-19 22:00 GMT+07:00 Fred Delrieu <notifications@github.com>:

> @thandonguocmo2020 <https://github.com/thandonguocmo2020> read README.md
> : https://github.com/caouecs/Laravel-lang/blob/master/README.md
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/caouecs/Laravel-lang/pull/512#issuecomment-211966128>
>



-- 
Thandonguocmo2020@gmail.com

**Pull request merged**